stdlib/3/configparser.pyi

```diff
@@ -102,9 +102,8 @@ class ConfigParser(_parser):
 
     # this signature is here only to placate the type checker, as a MutableMapping[str, _section]
 
-    # should be def get(self, section: str, option: str, *, raw: bool = ..., vars: _section = ..., fallback: str = ...) -> str: ...
-    # but it is incompatible with MutableMapping
-    def get(self, *args, **kwargs):  # type: ignore
+    # type: ignore since this is incompatible with MutableMapping
```

Comment:
Please don't start this comment with *exactly* `# type: ignore` since that may trigger other tools looking for `# type: ignore` on a new line. Just prefix with "Uses" or something.